### ISSUE TITLE
Keyboard focus indicator not visible in homepage body content

### ISSUE THREAD

**Issue by Lealikeintea** [istio/istio.io#7425]
When navigating through istio.io via keyboard using the tab key, the focus indicator is not visible in  the main body content on the homepage, violating two WCAG accessibility principles (linked below). 

[2.4.7 - Focus Visible (Level AA)](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0#focus-visible) - Any keyboard operable user interface has a mode of operation where the keyboard focus indicator is visible.

[2.4.8  - Location (Level AAA)](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0#location) - Information about the user's location within a set of Web pages is available.
